src/index.js

```diff
@@ -130,6 +138,10 @@ class Backend {
             i18nextElectronBackend
         } = this.backendOptions;
 
+        if (typeof i18nextElectronBackend === "undefined"){
```

Comment:
Instead of this, can you make an else for the `if` on line 122, and within this else put `this.setupIpcBindings();`?

Can you also put a comment above that line saying "Because we have the i18nextElectronBackend, we know we are in the renderer process and need to set up ipc bindings"

Reply:
Not sure to understand what you want to achieve here.
If you need to refactor your code, I guess it's best to do it outside this PR.

Reply:
I'd like to move the responsibility of the caller to determine whether or not to call `setupIpcBindings`, quite literally this:

```
// For cases we want to use this translator in the backend of Electron;
// ie menus and context menus, we need to be able to read/write translation
// files without using the window (it just doesn't exist!), so we dynamically
// import 'fs' here so that we are able to translate
if (typeof this.backendOptions.i18nextElectronBackend === "undefined") {
    this.fs = require("fs");
} else {

    // Because we have the i18nextElectronBackend, we know we are in the renderer process and need to set up ipc bindings
    this.setupIpcBindings();
}
```

instead of the line you had added here:
```
if (typeof i18nextElectronBackend === "undefined") {
    return;
}
```

If you want to just copy all the code I pasted above, that'd be great, otherwise I can simply make the change myself.

Reply:
I think it's better if you do it yourself, because I didn't add this line myself :
```
if (typeof i18nextElectronBackend === "undefined") {
    return;
}
```
I simply followed your direction and started from your commit https://github.com/reZach/i18next-electron-fs-backend/commit/1af4d358a260873b92b1af084dc15668660736f4

You did all the hard work and I simply had to fix a couple of little issues to make your changes works. I'm not confident to do some refactoring on your code and you have a better view than I have on your code.

Reply:
_Oh_! I forgot I even wrote this line - thanks for being patient with me here. As you can see it's obviously been a lot of time since I've looked at this.